src-input/duk_bi_json.c

```diff
@@ -1423,7 +1423,11 @@ DUK_LOCAL void duk__enc_fastint_tval(duk_json_enc_ctx *js_ctx, duk_tval *tv) {
 	 * "long long" type exists.  Could also rely on C99 directly but that
 	 * won't work for older MSVC.
 	 */
+#if defined(__MINGW32__) || defined(__MINGW64__)
+	DUK_SPRINTF((char *) buf, "%I64d", (long long) v);
+#else
 	DUK_SPRINTF((char *) buf, "%lld", (long long) v);
+#endif
```

Comment:
Strictly speaking, `long long` is not guaranteed to be 64-bit.  In practice, it *is* on most platforms, but I believe the C/C++ specifications only require `char` >= `short` >= `int` >= `long` >= `long long` without specifying minimum/maximum sizes for any of them (except `char` which is always 1 byte).